app/src/androidTest/java/net/skyscanner/backpack/fab/BpkFabTest.kt

```diff
@@ -48,6 +49,9 @@ class BpkFabTest : BpkSnapshotTest() {
 
   @Test
   fun screenshotTestFab_CustomIcon_Disabled() {
+    if (Looper.myLooper() == null) {
```

Comment:
Why do we need it?

Reply:
as I mentioned in one of the other components there were some issues when running the tests individually